.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
Shouldn't this have an empty new line? And perhaps a comment to explain what this does. Neat little function by the way.

Reply:
Regarding the end of line there's one, we have a hook to check that there is exactly one :)

Reply:
Huh that's weird. I thought Github would display those in "file-view" or in "raw-view" but I didn't see one so I assumed it was missing.

Thanks for the comment!